### DISCUSSION TITLE
Non Profit Use

### DISCUSSION THREAD

**Discussion by JMOCARRO** [makeplane/plane#6237] [General]
We have been using Plane to manage our projects for several months now and find it very useful. We are a small non-profit organization with a limited user group using the free online version. Are there any plans to offer the full featured online version for free to groups like us? We are concerned about fully committing to the project without knowing the future roadmap. Thanks.

**Comment by danciaclara** [comment]
For pricing-related queries, please feel free to email us at sales@plane.so, and we'll be happy to assist you
